bodhi/server/templates/override.html

```diff
@@ -138,6 +138,7 @@ <h2 class="pull-left">New Override</span>
 
     <input type="hidden" name="csrf_token" value="${request.session.get_csrf_token()}"/>
 
+    %if request.user and request.user.name is override.submitter.name:
```

Comment:
This check is more restrictive than the handler for these buttons, as the handler allows anyone in the packager group:

https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/services/overrides.py#L156